SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKDevAuthTokenGenerator.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Templates/Atomic.h"
+#include "Widgets/Notifications/SNotificationList.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialGDKDevAuthTokenGenerator, Log, All);
+
+class SPATIALGDKEDITOR_API FSpatialGDKDevAuthTokenGenerator : public TSharedFromThis<FSpatialGDKDevAuthTokenGenerator>
+{
+public:
+	FSpatialGDKDevAuthTokenGenerator();
+
+	void AsyncGenerateDevAuthToken();
+
+private:
+	void ShowTaskStartedNotification(const FString& NotificationText);
```

Comment:
Isn't this duplicate code of what's in SpatialGDKEditorToolbar?
Perhaps we should refactor the notification helpers into `SpatialGDKServices` as that's the topmost module?

Reply:
Hmm so it's possible to do that but we may need the different notifications at the same time, so the interface would be a bit more involved.

Reply:
I'd be fine with a follow-up task on notification management in the GDK

Reply:
https://improbableio.atlassian.net/browse/UNR-3533